docs/basics/detach_and_gdb.md

```diff
@@ -0,0 +1,51 @@
+---
+icon: material/location-exit
+search:
+    boost: 4
+---
+# :material-location-exit: Detach and GDB Migration
+
+In **libdebug**, you can detach from the debugged process and continue execution with the `detach()` method.
+
+!!! ABSTRACT "Function Signature"
+    ```python
+    d.detach()
+    ```
+
+!!! WARNING "Detaching from a running process"
+    Remember that detaching from a process is meant to be used when the process is stopped. If the process is running, the command will wait for a [stopping event](../../stopping_events/stopping_events). To forcibly stop the process, you can use the `interrupt()` method before migrating.
+
+## :simple-gnu: GDB Migration
+If at any time during your script you want to take a more traditional approach to debugging, you can seamlessly switch to [GDB](https://www.sourceware.org/gdb/). This will temporarily detach libdebug from the program and give you control over the program using GDB. Quitting GDB will return control to libdebug. 
```

Comment:
The API has been changed, review this part please

Reply:
[9e99adc](https://github.com/libdebug/libdebug/pull/129/commits/9e99adc7c4f740849576a6ec29d1c746c4e2a5ef)